Prace_domowe/Praca_domowa4/grudzien_adrianna/pd4.ipynb

```diff
@@ -0,0 +1,1231 @@
+{
```

Comment:
<p>RandomizedSearchCV działa "jak model". Tzn. ma fit i predict, czyli jak wytrenowałaś go 2 komórki wyżej, to wystarczy zrobić predict. Nie trzeba jeszcze raz fitować z tymi parametrami</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>